docs/using-scylla/workload-attributes.rst

```diff
@@ -0,0 +1,132 @@
+=============================
+Defining Workload Attributes
```

Comment:
@tzach In this article, I focused on workloads because the presentations during the summit implied that this is why the feature was added. 

"Service Level Attributes Properties" is a four-noun phrase, which makes it unclear. Also, it doesn't indicate what the user may want to achieve (our documentation should be more goal-oriented).
To make the title less workload-related, we could change it to "**Defining Service Level Attributes**".
What do you think?